models/public/rexnet-v1-x1.0/model.yml

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  ReXNet V1 x1.0 is image classification model pretrained on ImageNet dataset. Rank
+  eXpansion Network (ReXNet) is a network derived from research to mitigate the representational
+  bottleneck in a network.
+
+  The model input is a blob that consists of a single image of "1x3x224x224" in RGB
+  order.
+
+  The model output is typical object classifier for the 1000 different classifications  matching
+  with those in the ImageNet database.
+
+  For details see repository <https://github.com/clovaai/rexnet> and paper <https://arxiv.org/pdf/2007.00992.pdf>.
+task_type: classification
+files:
+  - name: rexnetv1_1.0x.pth
+    size: 19428695
+    sha256: 30b1700f2cda11f6d07eb1c414bab30879b8eb0d8c6571254fc3e65bf911ee46
+    source:
+      $type: google_drive
+      id: 1xeIJ3wb83uOowU008ykYj6wDX2dsncA9
+  - name: rexnetv1.py
+    size: 6335
+    sha256: 26ed1b143947f2d10198c7dba515c313719400daf05ea1efea434d8e61779331
+    source: https://raw.githubusercontent.com/clovaai/rexnet/104f2184754d97b8050f24bdf39f650b3e80cad8/rexnetv1.py
+postprocessing:
+  - $type: regex_replace
+    file: rexnetv1.py
+    pattern: 'USE_MEMORY_EFFICIENT_SWISH = True'
+    replacement: 'USE_MEMORY_EFFICIENT_SWISH = False'
+framework: pytorch
+conversion_to_onnx_args:
+  - --model-path=$dl_dir
+  - --model-name=ReXNetV1
+  - --weights=$dl_dir/rexnetv1_1.0x.pth
+  - --import-module=rexnetv1
+  - --model-param=width_mult=1.0
+  - --input-shape=1,3,224,224
+  - --output-file=$conv_dir/rexnet-v1-x1.0.onnx
+  - --input-names=input.1
+  - --output-names=1424
```

Comment:
I think named output is better that number of node

Reply:
Done